nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -171,6 +171,14 @@ let
           quic_bpf on;
         ''}
 
+        ${optionalString cfg.experimentalZstdSettings ''
```

Comment:
In line 253 we still have recommendedZstdSettings which will probably break eval and I don't think that the move of the setting was intentional.